recipes/wimpy/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "wimpy" %}
+{% set gh_author = "wimglenn" %}
+{% set version = "0.6" %}
+{% set sha256 = "014d21a31980eab48b92c4210e9fdba643b79f7081c64ea9bc97f2be9db9bd58" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ gh_author }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - wimpy
+
+about:
+  home: https://github.com/wimglenn/wimpy
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
There is no need for this field any more as it's computed automatically:

```suggestion
```

Reply:
After I removed the line `license_file: LICENSE` for both of them the bot is now complaining. Is it ok?